docs/setup/_installation_or_upgrade.mdx

```diff
@@ -0,0 +1,25 @@
+import GreenCheckMark from '@site/static/img/imported/graphics/green-check-icon-cc0v1.webp';
+import ScDocusaurusConfig from "@site/ScDocusaurusConfig.js";
+
+#### Step 1
+
+Open an FTP connection to the target Ewon device.
+
+#### Step 2
+
+Navigate to the /usr folder on the Ewon device's file system.
+
+#### Step 3
+
+Upload the downloaded connector files to the /usr folder.
+
+- <>{ScDocusaurusConfig.applicationJarFileName} where X.Y.Z are the connector version numbers</><br/>
+- `jvmrun`
+- <>{ScDocusaurusConfig.configFileName}</><br/>
+
```

Comment:
The step about certificates was removed here as its not applicable for all connectors. Perhaps there should be a note here to references a certificate section if applicable.

Reply:
I agree that we can accept this and add an issue for improvements. 
It's possible to have conditionals in MDX like an [IIFE](https://mdxjs.com/docs/what-is-mdx/#expressions) using braces, but I'm not sure if this can return markdown, or needs to return components or text 
```javaScript
{ useCerts === true ?   return "## heading about certs ..."  : return ""  }
```

}
```

Reply:
Yeah, I agree. I have approved/accepted this PR in its current state and the certificate documentation piece can be addressed in a future improvement.

I just quickly tested, and the markdown was interpreted as plain text. I think if we make it into a separate component/JS file, we can return `<p>` blocks with the text and use conditionals to include/exclude additional sections.